ENTITY/PlayEntityAnim.md

```diff
@@ -5,25 +5,20 @@ ns: ENTITY
 
 ```c
 // 0x7FB218262B810701 0x878753D5
-BOOL PLAY_ENTITY_ANIM(Entity entity, char* animName, char* animDict, float p3, BOOL loop, BOOL stayInAnim, BOOL p6, float delta, Any bitset);
-```
-
-```
-delta and bitset are guessed fields. They are based on the fact that most of the calls have 0 or nil field types passed in.  
-The only time bitset has a value is 0x4000 and the only time delta has a value is during stealth with usually <1.0f values.  
+BOOL PLAY_ENTITY_ANIM(Entity entity, char* animName, char* animDict, float fBlendDelta, BOOL bLoop, BOOL bHoldLastFrame, BOOL bDriveToPose, float fStartPhase, cs_type(Any) int nFlags);
```

Comment:
```suggestion
BOOL PLAY_ENTITY_ANIM(Entity entity, char* animName, char* animDict, float fBlendDelta, BOOL bLoop, BOOL bHoldLastFrame, BOOL bDriveToPose, float fStartPhase, int iFlags);
```

Int types don't need to use cs_types, also `nFlags` doesn't make sense here.

Reply:
Fixed the type, what do you mean with "nFlags doesn't make sense here"?

Reply:
oh wait i saw what you mean disregard